requirements.txt

```diff
@@ -0,0 +1,53 @@
+# This file was autogenerated by uv via the following command:
```

Comment:
Is this file needed in addition to `uv.lock`?

Reply:
@terrytangyuan in an ideal world, this file is not needed at all. but many folks still instinctively look for this file and have processes dependent on it. that's why `uv` even has this export mechanism.

Reply:
Sounds good